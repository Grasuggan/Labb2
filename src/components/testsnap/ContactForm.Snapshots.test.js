import React from "react";
import Greeting from "../Greeting.js";
import renderer from "react-test-renderer";


it("should match Greet snapshot", () => {
    const greet = renderer.create(  
         <Greeting first="Test" last="Testsson"/>
    );
    expect(greet).toMatchSnapshot();
});