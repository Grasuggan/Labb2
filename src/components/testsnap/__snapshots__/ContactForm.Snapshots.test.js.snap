// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match Greet snapshot 1`] = `
<div>
  <h1>
    Welcome 
    Test
     
    Testsson
  </h1>
</div>
`;
